pandas/tests/frame/methods/test_to_dict.py

```diff
@@ -513,6 +513,17 @@ def test_to_dict_masked_native_python(self):
         result = df.to_dict(orient="records")
         assert isinstance(result[0]["a"], int)
 
+    def test_to_dict_tight_no_warning_with_duplicate_column(self):
+        # GH#58281
+        df = DataFrame([[1, 2], [3, 4], [5, 6]], columns=["A", "A"])
+        assert df.to_dict(orient="tight") == {
```

Comment:
Can you still assert that `df.to_dict(orient="tight")` doesn't raise a warning?

Also can you structure like

```
result = 
expected = 
assert result == expected
```

Reply:
How about this time? Am I understanding correctly?